charts/signoz/Chart.yaml

```
@@ -22,27 +22,7 @@ dependencies:
   - name: clickhouse
     repository: https://signoz.github.io/charts
     condition: clickhouse.enabled
-    version: 24.1.2
-  - name: k8s-infra
-    repository: https://signoz.github.io/charts
-    condition: k8s-infra.enabled
-    version: 0.11.7
-  - name: cert-manager
-    repository: https://charts.jetstack.io
-    condition: cert-manager.enabled
-    version: 1.9.1
-  - name: ingress-nginx
-    repository: https://kubernetes.github.io/ingress-nginx
-    condition: ingress-nginx.enabled
-    version: 4.3.0
-  - name: minio
-    repository: https://charts.min.io
-    condition: minio.enabled
-    version: 4.0.9
-  - name: keycloak
-    repository: https://charts.bitnami.com/bitnami
-    condition: keycloak.enabled
-    version: 10.1.5
+    version: 24.1.4
```

Can you bump this to `24.1.5`?
I have a small change that can go along with this. - https://github.com/SigNoz/charts/pull/478

```suggestion
    version: 24.1.5
```

---

Let's merge that change first!